src/KafkaFlow.Retry/Durable/Polling/QueueTracker.cs

```diff
@@ -1,130 +1,171 @@
+                    "PollingJob Scheduler scheduled",
                     new
                     {
-                        PollingId = this.retryDurablePollingDefinition.Id,
-                        CronExpression = this.retryDurablePollingDefinition.CronExpression,
+                        SchedulerId = this.schedulerId,
+                        PollingJobType = jobDataProvider.PollingDefinition.PollingJobType.ToString(),
+                        CronExpression = jobDataProvider.PollingDefinition.CronExpression,
                         ScheduleJob = scheduledJob.ToString()
                     });
             }
             catch (Exception ex)
             {
                 this.logHandler.Error(
-                    "PollingJob Scheduler Error",
+                    "PollingJob Scheduler ERROR scheduling",
                     ex,
                     new
                     {
-                        PollingId = this.retryDurablePollingDefinition.Id,
-                        CronExpression = this.retryDurablePollingDefinition.CronExpression
+                        SchedulerId = this.schedulerId,
+                        PollingJobType = jobDataProvider.PollingDefinition.PollingJobType.ToString(),
+                        CronExpression = jobDataProvider.PollingDefinition.CronExpression
                     });
             }
         }
 
-        internal void UnscheduleJob(CancellationToken cancellationToken = default)
+        private void StartScheduler(CancellationToken cancellationToken)
         {
-            this.logHandler.Info(
-                "PollingJob Unscheduler Started",
-                new
-                {
-                    TriggerKey = this.trigger.Key.ToString()
-                });
+            Guard.Argument(this.scheduler).Null(s => "Scheduler was already started. Please call this method just once.");
+
+            lock (internalLock)
+            {
+                StdSchedulerFactory fact = new StdSchedulerFactory();
+                fact.Initialize(new NameValueCollection { { "quartz.scheduler.instanceName", this.schedulerId } });
+                this.scheduler = fact.GetScheduler(cancellationToken).GetAwaiter().GetResult();
 
-            var unscheduledJob = this.scheduler
-                .UnscheduleJob(this.trigger.Key)
-                .GetAwaiter()
-                .GetResult();
+                this.logHandler.Info("PollingJob Scheduler acquired", new { SchedulerId = this.schedulerId });
+            }
 
-            this.logHandler.Info("PollingJob Unscheduler Finished",
-                new
-                {
-                    UnscheduledJob = unscheduledJob,
-                    TriggerKey = this.trigger.Key.ToString()
-                });
+            if (!this.IsSchedulerActive)
+            {
+                this.scheduler
+                    .Start(cancellationToken)
+                    .GetAwaiter()
+                    .GetResult();
```

Comment:
This statement is outside the lock block, perhaps we can make it and the method async?